### PR TITLE
topics: add MuSig2 implementations and newsletter 324 mention

### PR DESCRIPTION
**Pull request by jonatack**
to follow up on https://github.com/bitcoinops/bitcoinops.github.io/pull/1942#discussion_r1797642476

### CODE REVIEW

File: _topics/en/musig.md [hunk 1]
```diff
@@ -28,9 +28,19 @@ primary_sources:
...
+
+    - title: MuSig2 implementation in Libsecp256k1
+      link: https://github.com/bitcoin-core/secp256k1/issues/1479
+
 ## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
 ## true" to bold entry
 optech_mentions:
+  - title: Libsecp256k1 adds MuSig2 BIP340-compatible multisig as specified in BIP327
+    url: /en/newsletters/2024/10/11/#libsecp256k1-1479
+    date: 2024-10-11
```
Comment (bitschmidty) [inline _topics/en/musig.md:42]: I dont think the date is needed for the newsletter entries

File: _topics/en/musig.md [hunk 2]
```diff
@@ -28,9 +28,19 @@ primary_sources:
...
+    - title: Original MuSig2 implementation (experimental)
+      link: https://github.com/BlockstreamResearch/secp256k1-zkp
+
+    - title: MuSig2 implementation in Libsecp256k1
+      link: https://github.com/bitcoin-core/secp256k1/issues/1479
+
 ## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
 ## true" to bold entry
 optech_mentions:
+  - title: Libsecp256k1 adds MuSig2 BIP340-compatible multisig as specified in BIP327
```
Comment (bitschmidty) [inline _topics/en/musig.md:40]: Would you mind moving to the end (most recents)

File: _topics/en/musig.md (hunk 1 above)
Comment (jonatack) [reply _topics/en/musig.md:42]: Done

File: _topics/en/musig.md (hunk 2 above)
Comment (jonatack) [reply _topics/en/musig.md:40]: Done (thanks! oops, not sure how I didn't see that, probably am used to ordering by most recent first, as in https://keepachangelog.com/en/1.1.0/)

**Comment by jonatack** [comment]
Updated the mention commit per @bitschmidty feedback (thanks!)

**Review by bitschmidty — APPROVED** [review]
ACK, thank @jonatack

**Pull request merged**